docs/getting-started/index.mdx

```diff
@@ -6,174 +6,377 @@ pagination_next: http/index
+		<div className="ngrok--card ngrok--card-sm hover:bg-card-hover">
+			<div className="ngrok--card-heading jc-space-between">
+				<h3 className="fw-600">IP restrictions</h3>
+			</div>
+			<p>Allow or deny traffic based on the source IP address or CIDR range.</p>
+		</div>
+	</Link>
+	<Link to={`/docs/http/traffic-policy/actions/verify-webhook/`}>
+		<div className="ngrok--card ngrok--card-sm hover:bg-card-hover">
+			<div className="ngrok--card-heading jc-space-between">
+				<h3 className="fw-600">Webhook verification</h3>
+			</div>
+			<p>
+				Verify that HTTP requests from webhook providers are signed with a known
+				secret.
+			</p>
+		</div>
+	</Link>
+</div>
+
+## Step 5: Run ngrok persistently
+
+While this guide gets you started with ngrok, you might want your agent endpoint
+to keep running even after you close your terminal window, or to automatically
+start when your system boots up.
+
+<Tabs groupId="agent" queryString="agent-persistence">
+	<TabItem value="linux" label="Linux (systemd)" default>
+    
+    On Linux systems, you can create a `systemd` unit file that starts on boot,
+    restarts automatically, and logs to `syslog`.
+
+    You'll need to [create a configuration file](/docs/agent/config/), or [modify
+    the default](/docs/agent/config/index.mdx#default-locations), to contain
+    your authtoken and definitions of all the agent endpoints you want to
+    create. Then you can install the service with the CLI:
+
+    ```shell
+    ngrok service install --config "~/path/to/ngrok.yml"
+    ```
+
+    And start it:
+
+    ```shell
+    ngrok service start
+    ```
+
+    See our [background service docs](/docs/agent/index.mdx#background-service)
+    for details.
+
+    </TabItem>
+    <TabItem value="windows" label="Windows">
+
+    On Windows, you can create a Windows service, with errors and warnings
+    logged to the Windows event log.
+
+    You'll need to [create a configuration file](/docs/agent/config/), or [modify
+    the default](/docs/agent/config/index.mdx#default-locations), to contain
+    your authtoken and definitions of all the agent endpoints you want to
+    create. Then you can install the service with the CLI:
+
+    ```shell
+    ngrok service install --config "C:\path\to\ngrok.yml"
+    ```
+
+    And start it:
+
+    ```shell
+    ngrok service start
+    ```
+
+    See our [background service docs](/docs/agent/index.mdx#background-service)
+    for details.
+
+    </TabItem>
+
+  <TabItem value="docker" label="Docker">
+
+    Use our pre-built Docker images to containerize ngrok for easy deployment
+    and management. To create an agent endpoint for the same service used above
+    on `localhost:8080`, use the following:
+
+    ```shell
+      docker run --net=host -it -e NGROK_AUTHTOKEN=xyz ngrok/ngrok:latest http 8080
+    ```
+
+    See our [Docker docs](/docs/using-ngrok-with/docker/) for more details.
+
+  </TabItem>
+  <TabItem value="detach" label="Tmux/Screen/Zellij">
```

Comment:
this doesn't actually tell you how to run persistently on any of these systems. how do you detach it?